modules/debts_api/app/controllers/debts_api/v0/financial_status_reports_controller.rb

```diff
@@ -38,7 +38,9 @@ def submissions
       def rehydrate
         submission_id = params[:submission_id]
 
-        DebtsApi::V0::FsrRehydrationService.attempt_rehydration(user_uuid: current_user.uuid, submission_id:)
+        DebtsApi::V0::FsrRehydrationService.attempt_rehydration(user_uuid: current_user.uuid,
+                                                                user_account_id: current_user.user_account.id,
```

Comment:
Let's just pass in `current_user.user_account` it'll make the rest a bit simpler and remove a query downstream